kafka_consumer/datadog_checks/kafka_consumer/client/confluent_kafka_client.py

```diff
@@ -1,14 +1,76 @@
 # (C) Datadog, Inc. 2023-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+from confluent_kafka.admin import AdminClient
+from confluent_kafka import Consumer, TopicPartition
 
+EXCLUDED_TOPICS = ['__consumer_offsets', '__transaction_state']
 
 class ConfluentKafkaClient:
-    def __init__(self) -> None:
-        pass
+    def __init__(self, config, tls_context, log) -> None:
```

Comment:
Once https://github.com/DataDog/integrations-core/pull/14022 is merged, we can remove this `__init__()` since the parent class will have this.